src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const partnerName = (person.sex === 'f'
+    ? (`${person.partnerName} is my husband`)
+    : (`${person.partnerName} is my wife`));
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${person.name}`}
+      </h2>
+      {person.age
+        ? <p className="Person__age">{`I am ${person.age}`}</p>
+        : ''
+      }
```

Comment:
```suggestion
      {person.age
        && <p className="Person__age">{`I am ${person.age}`}</p>
      }
```
You can use && here 
https://react.dev/learn/conditional-rendering#logical-and-operator-